src/Nordea/Components/AccordionMenu.elm

```diff
@@ -0,0 +1,63 @@
+module Nordea.Components.AccordionMenu exposing (OptionalConfig(..), Status(..), content, header, view)
+
+import Css exposing (alignItems, center, display, displayFlex, float, inlineBlock, justifyContent, lastChild, listStyle, none, nthLastChild, rem, right, spaceBetween, width)
+import Css.Global exposing (children, typeSelector, withAttribute)
+import Html.Styled as Html exposing (Attribute, Html, div)
+import Html.Styled.Attributes exposing (attribute, css)
+import Nordea.Resources.Icons as Icons
+
+
+type Status
+    = Open
+    | Closed
+
+
+type OptionalConfig
+    = Status Status
+
+
+view : List OptionalConfig -> List (Attribute msg) -> List (Html msg) -> Html msg
+view config attrs children_ =
+    let
+        config_ =
+            config
+                |> List.foldl
+                    (\attr acc ->
+                        case attr of
+                            Status x ->
+                                { acc | status = x }
+                    )
+                    { status = Open }
+
+        openAttr =
+            case config_.status of
+                Open ->
+                    attribute "open" "" :: attrs
+
+                Closed ->
+                    attrs
+
+        attrs_ =
+            css
+                [ children [ typeSelector "summary" [ listStyle none, children [ typeSelector "div" [ lastChild [ display inlineBlock ], nthLastChild "2" [ display none ] ] ] ] ]
+                , withAttribute "open" [ children [ typeSelector "summary" [ children [ typeSelector "div" [ lastChild [ display none ], nthLastChild "2" [ display inlineBlock ] ] ] ] ] ]
+                ]
```

Comment:
May be easier to read with class names:
```suggestion
    css
        [ children [ typeSelector "summary" [ listStyle none, descendants [ class "left-arrow" [ display inlineBlock ], class "down-arrow" [ display none ] ] ] ]
        , withAttribute "open"
            [ children [ typeSelector "summary" [ descendants [ class "left-arrow" [ display none ], class "down-arrow" [ display inlineBlock ] ] ] ]
            ]
        ]
```

Reply:
I wanted to assign them a class like that but then in a use like this, if anyone actually uses two or three elements which belong to class `down-arrow` inside the `summary` tag, I thought it would be problematic. Are these names defined somewhere or can I come up with names like that?

Reply:
We very rarely use class names in elm (only for special cases like these), so I don't think it will be a problem. Most css is also scoped through elm-css